owl/EASE-OBJ.owl

```diff
@@ -144,6 +276,19 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#isModifiedBy -->
+
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#isModifiedBy">
+        <rdfs:subPropertyOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#isConstituentOf"/>
```

Comment:
Logically, this does not result in contradictions or concepts that are subconcepts of Nothing, so on that front it's ok.

On the other hand, this appears to insert a relational aspect "too soon" in the disposition model. Example:

I have a spraycan of paint. I can use it to alter the color of a wall, so presumably it will have some sort of Alteration disposition.

I think your intuition is, since I just described the spraycan by "I can alter the color of a wall", clearly I had to make reference to a modified quality in order to make sense of what the spraycan does.

My problem is, when I talk about the spraycan's disposition in general, I don't have a particular wall, with its particular Color quality individual, in mind. Yes, when we describe an event of actual painting, it will be useful to have a modifies/isModifiedBy relation, because the Event is about specific entities interacting.

However when discussing the disposition in the abstract, I don't want to depend on something conjectural and outside of it. That is, the constituency of the disposition of a spraycan cannot be the Color quality of a particular wall.

Reply:
No, my reasoning is rather: I want to express what aspect of the object is altered, so I need to refer to the quality type. That's what we were talking about earlier... where I said that we may need to use *QualityType* here to classify qualities. But that seems silly at the moment. It was hard enough to come up with one quality taxonomy :)

It's strange that DUL does not assert that a quality must be linked to a object holding it! So it would not make the ontology inconsistent if dispositions have their individual qualities, as far as I can tell.
I was also thinking about only using universal restrictions which would also not cause problems in the ABOX, and could be used for reasoning in the TBOX.

So, any suggestion how to do it?

Reply:
I guess the taxonomy needs to move to the concept branch